app/src/main/res/layout/dialog_donate_qr.xml

```diff
@@ -71,7 +71,7 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintEnd_toStartOf="@+id/layout_copy"
-            android:layout_marginVertical="@dimen/medium_margin"
+            android:layout_marginVertical="12dp"
```

Comment:
I believe the original dimension is `8dp`, unless we explicitly want to change the UI.